server/googleActions.js

```diff
@@ -185,7 +185,10 @@ const GoogleActions = {
 				'title'     : brew.title,
 				'views'     : '0',
 				'pageCount' : brew.pageCount,
-				'renderer'  : brew.renderer || 'legacy'
+				'renderer'  : brew.renderer || 'legacy',
+				'isStubbed' : true,
+				'createdAt' : new Date().toISOString(),
```

Comment:
Same thing here. We already utilize the built-in `obj.data.createdTime` here from the Google Drive file.